rosdep/python.yaml

```diff
@@ -6634,6 +6634,7 @@ python3-hypothesis:
   ubuntu: [python3-hypothesis]
 python3-ifcfg:
   debian:
+    bullseye: [python3-ifcfg]
     buster: [python3-ifcfg]
     stretch: [python3-ifcfg]
```

Comment:
I think that we should just change this to a generic key for all debian versions:

```
debian: [python3-ifcfg]
```

That will make it future-proof.  The same goes for the Ubuntu rule below.

Reply:
addressed at https://github.com/ros/rosdistro/pull/31482/commits/cdc26081043a7bd054869ab7b4caeca2daf0052d.